src/VisualStudio/Core/Def/Implementation/TodoComments/VisualStudioTodoCommentsService.cs

```diff
@@ -21,14 +21,20 @@
 using Microsoft.CodeAnalysis.PooledObjects;
 using Microsoft.CodeAnalysis.Remote;
 using Microsoft.CodeAnalysis.TodoComments;
+using Microsoft.VisualStudio.LanguageServices.ExternalAccess.VSTypeScript.Api;
 using Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem;
 using Roslyn.Utilities;
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.TodoComments
 {
     [Export(typeof(IVisualStudioTodoCommentsService))]
+    [Export(typeof(IVsTypeScriptTodoCommentService))]
```

Comment:
>IVsTypeScriptTodoCommentService [](start = 19, length = 31)

It'd be cleaner to export TS interface on a new, TS specific Service that imports VisualStudioTodoCommentsService and calls ReportTodoCommentDataAsync on it.

Reply:
As a follow up, also add VSTypeScriptTodoComment wrapper because TodoComment is not public.

---
In reply to: [399405480](https://github.com/dotnet/roslyn/pull/42821#discussion_r399405480) [](ancestors = 399405480)

Reply:
> As a follow up, also add VSTypeScriptTodoComment wrapper because TodoComment is not public.

How important is it to do that now though given that TS *already* uses TodoComment?